docs/api/types/chat_member.md

```diff
@@ -23,6 +23,7 @@ This object contains information about one member of a chat.
 | `can_invite_users` | `#!python Optional[bool]` | Optional. Administrators and restricted only. True, if the user is allowed to invite new users to the chat |
 | `can_pin_messages` | `#!python Optional[bool]` | Optional. Administrators and restricted only. True, if the user is allowed to pin messages; groups and supergroups only |
 | `is_member` | `#!python Optional[bool]` | Optional. Restricted only. True, if the user is a member of the chat at the moment of the request |
+| `is_chat_admin` | `#!python bool` | True if the user is administrator or creator of the chat |
```

Comment:
Aliases/additional methods/properties should not be mixed with original attributes in types because in this listed attributes from original Bot API documentation. Also this table can be overwrited by code-gen.

Just add after "Attributes" table new section named "Extensions" and then add the table with new methods and helper class.
Helper can be described in section "ChatMemberStatus Helper"